### DISCUSSION TITLE
How to customize Authorize Swagger form fields?

### DISCUSSION THREAD

**Discussion by skysantoroa** [fastapi/fastapi#4793] [Questions] [answered]
I'd like to define custom fields in the Authorize form and modify the way Swagger creates requests towards secured endpoints (by default adding the Authorization bearer)

![image](https://user-images.githubusercontent.com/101178279/163179332-ade40a0d-5a0a-493c-821b-acef91a80afe.png)

I defined new classes by extending the default ones without success, is that even possible?

**Comment by ghost** [comment]
I also have same issue... I do have two fields for login,

1. providerName
2. providerId

This is for a kinda ssoLogin api... But it doesn't working with swagger ui but working with postman client...

**Comment by YuriiMotov** [comment, marked as answer]
Swagger doesn't use the schema of your token endpoint to generate this authorization form.
This form is following the standard and just hardcoded in Swagger UI code.
So, to change it you will probably need to self-host Swagger UI docs and modify its assets
